app/src/main/java/ca/pkay/rcloneexplorer/Services/SyncService.java

```diff
@@ -19,9 +19,14 @@
 import androidx.localbroadcastmanager.content.LocalBroadcastManager;
 import androidx.preference.PreferenceManager;
 
+import org.json.JSONException;
+import org.json.JSONObject;
+
 import java.io.BufferedReader;
 import java.io.IOException;
 import java.io.InputStreamReader;
+import java.math.RoundingMode;
```

Comment:
Unused import - I think I know what you tried, see comment where progress percent is calculated

Reply:
removed